lisa/wlgen/rta.py

```diff
@@ -826,16 +834,20 @@ def get_rtapp_repr(self, task_name, plat_info):
         else:
             nodes_membind = self.numa_nodes_membind
         phase['nodes_membind'] = nodes_membind
-            
+
         return phase
 
 class RTATask:
     """
     Base class for conveniently constructing params to :meth:`RTA.by_profile`
 
-    :param sched_policy: the scheduler policy for this task.
+    :param sched_policy: the scheduler policy for this task. Defaults to
+      ``SCHED_OTHER``, see ``man sched`` for information on scheduler policies.
```

Comment:
Maybe we can start using the manpage role ?
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-manpages_url

Reply:
Done!